Makefile

```diff
@@ -1,22 +1,36 @@
-all: pycaqtimage.so
-SIPTAG    = Qt_4_6_2
-SIPINC    = $(PSPKG_RELDIR)/share/sip
-PYTHONINC = $(PSPKG_RELDIR)/include/python2.7
-SIPHDIR   = $(PSPKG_RELDIR)/include/python2.7
-QTINC     = $(PSPKG_RELDIR)/include
-QTLIB     = $(PSPKG_RELDIR)/lib
-
-pycaqtimage.so: pycaqtimage_sip_wrap.sip
-	sip -t $(SIPTAG) -t WS_X11 -I$(SIPINC) \
-	    -e -j 1 -c . pycaqtimage_sip_wrap.sip
-	g++ -g -I$(PYTHONINC) -I$(SIPHDIR) -I$(QTINC) -I$(QTINC)/QtCore -I$(QTINC)/QtGui \
-	    -fno-strict-aliasing -fPIC -D_REENTRANT -D__pentium__ -Wall -O4 -m64 \
-	    -c sippycaqtimagepart0.cpp
-	g++ -g -m64 -shared -L$(QTLIB) -lQtGui \
-	    sippycaqtimagepart0.o -o pycaqtimage.so
+SHELL = /bin/bash
+ENV = export PCDS_CONDA_VER=5.1.1; source /cds/group/pcds/pyps/conda/pcds_conda;
+ALL = pycaqtimage/pycaqtimage.so camviewer_ui.py advanced_ui.py markers_ui.py \
+      specific_ui.py timeout_ui.py icon_rc.py 
+
+all: $(ALL)
+
+pycaqtimage/pycaqtimage.so: pycaqtimage/configure.py pycaqtimage/pycaqtimage.sip
+	$(ENV) python pycaqtimage/configure.py pycaqtimage
+	$(ENV) make -C pycaqtimage
+
+camviewer_ui.py: camviewer.ui
+	$(ENV) pyuic5 -o camviewer_ui.py camviewer.ui
```

Comment:
We tend to just load these `.ui` files up dynamically via `uic.loadUiType` (e.g., https://github.com/slaclab/pydm/blob/e904d493524b70310cbe4ac6491a8092dbc120a6/pydm/display.py#L71) but your way is definitely more runtime performant

Reply:
Yeah, this wasn't my codebase originally.  I'm not crazy about the naming here either.  (*ui_impl.py?)